Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -645,7 +645,12 @@ namespace AzToolsFramework
         m_handlerCleanupTimer->setInterval(0);
         connect(m_handlerCleanupTimer, &QTimer::timeout, this, &DocumentPropertyEditor::CleanupReleasedHandlers);
 
-        m_spawnDebugView = ed_showDPEDebugView;
+        if (auto* console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
```

Comment:
Do we actually need the `console != nullptr` ? Doesn't the value already get tested in the `if` as a result of the assignment operator?

Reply:
The left side is using the C++17 [if initialization](https://en.cppreference.com/w/cpp/language/if) block, it is not checking the condition,

Reply:
yep! What Derric said :)